index.html

```diff
@@ -63,14 +63,306 @@ <h1 class="hero-title">Want anything to be easy with LaslesVPN.</h1>
               Provide a network for all your needs with ease and fun using
               LaslesVPN discover interesting features from us.
             </p>
-            <button class="rectang-button">Get Started</button>
+            <button class="button-secondary">Get Started</button>
           </div>
           <div>
             <img src="images/illustration-hero.svg" alt="" />
           </div>
         </div>
       </section>
+      <section class="features-section">
+        <div class="container">
+          <div class="stats-wrapper">
+            <ul class="d-flex justify-content-between">
+              <li class="align-items-center d-flex">
+                <div class="stats-icon-wrap d-flex">
```

Comment:
Let's get rid of `d-flex` here. Use `stats-icon-wrap` to add necessary styles. Please check for entire document.